tests/unit/dataloader/test_tf_dataloader.py

```diff
@@ -80,7 +66,9 @@ def test_simple_model():
     outputs = tf.keras.layers.Dense(1, activation="softmax")(outputs)
     model = tf.keras.Model(inputs=inputs, outputs=outputs)
     model.compile(optimizer="sgd", loss="binary_crossentropy", metrics=["accuracy"])
-    model.fit(loader, epochs=2)
+    history_callback = model.fit(loader, epochs=2, shuffle=False)
+    assert len(history_callback.history["loss"]) == 2
```

Comment:
Reverting sounds good to me, but I'm just wondering if this assertion will catch what we are expecting to catch? Because as I understand it, keras will still run the second epoch but it will just not have any data from the generator.